ucg/flows/product-store/readme.md

```diff
@@ -12,23 +12,26 @@ Certain settings are essential for every Product Store to ensure efficient opera
 
 | **Field**                  | **Description**                                                                                                       | **Value**             |
 |------------------------|---------------------------------------------------------------------------------------------------------------------|--------------------|
-| Store Name             | Product Store serves as a representation of a company or brand, particularly useful for retailers managing multiple brands. Make sure to specify the brand name before proceeding. | Frank and Oak      |
-| Inventory Facility      | Default facility used to push inventory counts from HotWax Commerce to eCommerce                                   | _NA_               |
+| Store Name             | Product Store serves as a representation of a company or brand, particularly useful for retailers managing multiple brands. Make sure to specify the brand name before proceeding. | Frank And Oak      |
+| Inventory Facility      | Default facility used to push inventory counts from HotWax Commerce to eCommerce                                   | \_NA_               |
```

Comment:
why are we adding a back slash here?

Reply:
The facility name in oms configs is "_NA_" 
The syntax in markdown considers text between two underscores to be displayed as italic, in preview and on gitbook frontend the back slash will not be visible because it's purpose is to only negate the italic syntax

Reply:
got it. thanks!